### ISSUE TITLE
Importing library on server side causes crash

### ISSUE THREAD

**Issue by nealoke** [jpuri/draftjs-utils#30]
When importing this library I get the error that the `window` is not defined as it is rendered on the server side.

In the js folder of this project the window is not used but when checking out the `lib/draftjs-utils.js` I see that there is a reference to the window. It would be nice if you could alter it so it doesn't crash on SSR 😢

**Comment by sedenardi** [comment]
This issue is present in any version after `0.9.4`.

**Comment by eseQ** [comment]
Yep, cause webpack build `umd` and using `window` for global object
In `lib/draftjs-utils.js`
```javascript
!function(t,e){
  "object"==typeof exports&&"object"==typeof module
    ? module.exports=e(require("draft-js"),require("immutable"))
    : "function"==typeof define&&define.amd
      ? define(["draft-js","immutable"],e)
      :"object"==typeof exports
        ? exports.draftjsUtils=e(require("draft-js"),require("immutable"))
        : t.draftjsUtils=e(t["draft-js"],t.immutable)
}(window, function(){...})
```

Answer is set `output.globalObject` to `this` in [webpack.prod.config.js](https://github.com/jpuri/draftjs-utils/blob/master/config/webpack.prod.config.js#L11). Documentation [here](https://webpack.js.org/configuration/output/#outputglobalobject). Discussion [here](https://github.com/webpack/webpack/issues/6642).

**Comment by eseQ** [comment]
Awesome. After half of year I have to check it issue again. And I wrote answer for this already

**Comment by sedenardi** [comment]
> Awesome. After half of year I have to check it issue again. And I wrote answer for this already

@eseQ To be clear, that's what `draftjs-utils` needs to do to solve this, users of the library can't add this to their own config to eliminate this issue right?

**Comment by eseQ** [comment]
@sedenardi correct. #39 PR was created by me one year ago )
That about build `draftjs-utils` package not about user `webpack`'s configuration

**Comment by sedenardi** [comment]
Thanks for your help. Unfortunate that this library seems abandoned at this point.

**Comment by eseQ** [comment]
Yeap =( But you can fork this library and make the changes.
I have stopped working on draftjs part for my project's

**Comment by eseQ** [comment]
And one more moment: you can easy solve that issue on your side: just load you component with draft-editor (when using `draftjs-utils` only for client side (use `{ ssr: false }` for [`loadable-components`](https://loadable-components.com/docs/api-loadable-component/#loadable) for example)

**Comment by sedenardi** [comment]
> Yeap =( But you can fork this library and make the changes. I have stopped working on draftjs part for my project's

Out of curiosity, what library have you been using instead of draftjs?

**Comment by eseQ** [comment]
I using draftjs - is awesome. Just don't work on it now. I mean have no time to fork and work on this library
